flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -234,16 +234,15 @@
 	/**
 	 * Whether TaskManager managed memory should be pre-allocated when the TaskManager is starting.
 	 */
+	@Deprecated
```

Comment:
True, if it is configured it will be just ignored. I wanted to leave it for the documentation purposes but the better place is release notes. I will remove it.